src/app/footer/footer.component.ts

```diff
@@ -59,7 +61,12 @@ export class FooterComponent extends Base implements OnInit {
       .subscribe(
         (metadata: Metadata) => {
           if (metadata.hasOwnProperty('version')) {
-            this.version = metadata['version'];
+            const metadatum = metadata['version'];
+            if ((metadatum && metadatum.indexOf('SNAPSHOT') !== -1) || metadatum.indexOf('development-build') !== -1) {
```

Comment:
i feel like there's another alternative to indexOf !== -1

Reply:
Hmm, did not know. There is `includes`, which is not supported on IE, but I guess target es5 should address that. We use it in other places, so updated it.